Source/ARTAuth.m

```diff
@@ -275,13 +275,13 @@ - (void)executeTokenRequest:(ARTTokenRequest *)tokenRequest callback:(void (^)(A
 
 - (void)authorise:(ARTTokenParams *)tokenParams options:(ARTAuthOptions *)authOptions callback:(void (^)(ARTTokenDetails *, NSError *))callback {
     BOOL requestNewToken = NO;
-    ARTAuthOptions *mergedOptions = [self mergeOptions:authOptions];
-    [self storeOptions:mergedOptions];
-    ARTTokenParams *currentTokenParams = [self mergeParams:tokenParams];
```

Comment:
I believe timestamp and queryTime was not carried forward before, but now it appears they are given these objects are blindly copied?  See http://docs.ably.io/client-lib-development-guide/features/#RSA10g